test/webpack.config.js

```diff
@@ -89,7 +83,10 @@ const TEST_CONFIG = Object.assign({}, COMMON_CONFIG, {
   },
 
   plugins: [
-    new HtmlWebpackPlugin({title: 'loaders.gl tests'})
+    new HtmlWebpackPlugin({title: 'loaders.gl tests'}),
+    new webpack.DefinePlugin({
```

Comment:
Where is `TEST_DATA_DIR` used? 

Node tests run without webpack, do they get access to this constant?

Reply:
This was left over from something else I was testing. Will remove.